src/BinSkim.Rules/DwarfRules/BA3003.EnableStackProtector.cs

```diff
@@ -43,9 +48,11 @@
~        {
~            CanAnalyzeDwarfResult result = default;
```

Comment:
>elf [](http://example.com/codeflow?start=36&length=3)

Please fix LGTM alert:
1. elf is not needed assign since now it is not used.

Reply:
fixed